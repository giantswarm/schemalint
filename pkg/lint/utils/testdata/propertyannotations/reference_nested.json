{
  "properties": {
    "person": {
      "$ref": "nested.json"
    }
  },
  "type": "object"
}
