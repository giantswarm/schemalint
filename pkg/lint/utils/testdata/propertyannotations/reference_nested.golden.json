{
  "/person": {},
  "/person/addresses": {
    "Title": {
      "Value": "Addresses",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The person's addresses.",
      "ReferenceLevel": 1
    }
  },
  "/person/addresses/items/city": {
    "Title": {
      "Value": "City",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The city.",
      "ReferenceLevel": 1
    }
  },
  "/person/addresses/items/state": {
    "Title": {
      "Value": "State",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The state.",
      "ReferenceLevel": 1
    }
  },
  "/person/addresses/items/street": {
    "Title": {
      "Value": "Street",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The street address.",
      "ReferenceLevel": 1
    }
  },
  "/person/addresses/items/zip": {
    "Title": {
      "Value": "Zip",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The zip code.",
      "ReferenceLevel": 1
    }
  },
  "/person/age": {
    "Title": {
      "Value": "Age",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "Age in years which must be equal to or greater than zero.",
      "ReferenceLevel": 1
    }
  },
  "/person/firstName": {
    "Title": {
      "Value": "First Name",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The person's first name.",
      "ReferenceLevel": 1
    }
  },
  "/person/lastName": {
    "Title": {
      "Value": "Last Name",
      "ReferenceLevel": 1
    },
    "Description": {
      "Value": "The person's last name.",
      "ReferenceLevel": 1
    }
  }
}
